docs/tre-workspace-authors/firewall-rules.md

```diff
@@ -1,128 +1,256 @@
-# Adding Firewall Rules as part of a workspace or service deployment
+# Configuring Firewall Rules in Azure TRE
 
-!!! note
-    Creating firewall rules will in the future only be allowed through the firewall shared services [#882](https://github.com/microsoft/AzureTRE/issues/882).
+This guide explains how to configure firewall rules for your Azure TRE environment.
 
-A TRE service may require certain firewall rules to be opened in the TRE firewall. Examples include:
+## Contents
+- [Overview](#overview)
+- [Types of Firewall Rules](#types-of-firewall-rules)
+- [Configuring Application Rules](#configuring-application-rules)
+- [Configuring Network Rules](#configuring-network-rules)
+- [Applying Your Configuration](#applying-your-configuration)
+- [Examples](#examples)
 
-- Access to an external authorisation endpoint
-- Access to an external data store
-- Access to an external API
+## Overview
 
-Please be aware when opening firewall rules there is the potential for data to be leaked from the workspace to the external location.
+The Azure TRE Firewall shared service controls network traffic flowing in and out of your research environment. The firewall configuration is defined using a JSON file that specifies rule collections and their associated rules.
 
-## Using Terraform to open firewall rules
+## Types of Firewall Rules
 
-Until a mechanism to update shared services has been implemented, firewall rule updates should be done using terraform as part of the service deployment. The aim is to create a firewall rule that grants access from the workspace's address space to the external location. A challenge with this is that the rule must use a priority that has not been used by any other rule.
+The Azure TRE Firewall supports two types of rules:
```

Comment:
Again I would say two types or rules can be defined as part of the deployment of templates in `template_schema.json`